### ISSUE TITLE
MPI Parallelize training of Bayesian Polishing

### ISSUE THREAD

**Issue by biochem-fan** [3dem/relion#685]
This can be trivially parallelized by MPI.

We can calculate target function values per movie over MPI and gather it to the rank 0 node to run optimization.

**Comment by biochem-fan** [comment]
Jasenko is working on this.

**Comment by zruan** [comment]
I agree. This would be very helpful to speed up the training step. Any estimate when this feature will be implemented?

**Comment by biochem-fan** [comment]
I have no idea.
